verification/curator-service/api/src/model/parser.ts

```diff
@@ -3,8 +3,7 @@ import mongoose from 'mongoose';
 export const parserSchema = new mongoose.Schema({
     awsLambdaArn: {
         type: String,
-        required: 'Enter a parser AWS Lambda ARN',
-        match: [/^arn\:aws\:lambda\:.+\:.+\:function\:.+/],
+        required: 'Enter a parser AWS Lambda ARN or function name',
```

Comment:
It'd be good to reference the documentation that describes what a valid function name is, here and/or in the OpenAPI spec (e.g., meaning it can be ARN, semantic name, perhaps something else).

Reply:
I don't there there is a spec for function names, I added a real example in the openapi spec already but AFAICT this name is built by SAM based on the stuff we put in the template and the project overall (prefix is fixed, then function name from template, then random digitsletters)